doc/hands_on/hands_on_parquet.md

```diff
@@ -0,0 +1,240 @@
+# Hands-on S3-Virtual-Schema With Parquet Files
+
+## Introduction
+
+In this guide, we will walk through the setup of the Virtual Schema for document files stored on [AWS S3](https://aws.amazon.com/s3/). As an example we will create a virtual-schema for a parquet file.
+
+There is also a different [tutorial using JSON files](./hands_on.md).
+
+You need:
+
+* An Exasol database
+* An AWS account
+
+## What is This for?
+
+In this tutorial, we will create a virtual schema for Parquet files stored in a S3 Bucket. If you've never heard of S3, that is not a problem. You can imagine it as a very basic file storage. You can store files and access them using a key.
+
+We are going to create a Virtual Schema for these files so that you can access them just like a regular Exasol table.
+
+The example file we use in this tutorial has the same content as the three JSON files in the [books/](./books) folder. To get a first impression you can take a look on them instead of the parquet file itself, since for the parquet file you need a viewer.
```

Comment:
```suggestion
The example file we use in this tutorial has the same content as the three JSON files in the [books/](./books) folder. To get a first impression you can take a look at them instead of the parquet file itself since for the parquet file you need a viewer.
```